docs/reference/nuget-config-file.md

```diff
@@ -218,7 +218,7 @@ Stores keys for sources that use API key authentication, as set with the [`nuget
 
 ### disabledPackageSources
 
-Identified currently disabled sources. May be empty.
+Identified currently disabled sources. May be empty. Unless specific sources are disabled in this section, they are enabled.
```

Comment:
Is this always true? A package source can be disabled in other ways. For example, a package source from my shared nuget.config file can be disabled by adding a project specific nuget.config file that has `<clear />` in its `<packageSources>` node.

Reply:
That is more of a behavior of clear & merging of config files than it is with this section right?

Reply:
May be the text `section` is throwing different meanings. We can live without this line I guess and explain in the example?
Or just state:
Unless specific sources are disabled using `disabledPackageSources` , they are enabled.